services/enqueueing/enqueueing.go

```diff
@@ -0,0 +1,45 @@
+package enqueueing
+
+import (
+	"errors"
+
+	"google-scraper/database"
+	"google-scraper/models"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocraft/work"
+	"github.com/beego/beego/v2/server/web"
+)
+
+var enqueuer *work.Enqueuer
+
+func init() {
+	enqueuer = work.NewEnqueuer("google_scraper", database.GetRedisPool())
+}
+
+func EnqueueKeywordJob(keyword *models.Keyword, throttleMultiplier int64) (*work.ScheduledJob, error) {
+	if keyword.Id <= 0 {
+		return nil, errors.New("invalid keyword object")
```

Comment:
Can we add more additional information on why this return an error? I think it would be easier for debugging in the future.